.teamcity/Ribasim/buildTypes/Ribasim_Ribasim_MakeGitHubRelease.xml

```diff
@@ -13,27 +13,9 @@
 set -euxo pipefail
 . /usr/share/Modules/init/bash
 
-module load github
-# Get the name of the currently checked out tag
-tag_name=$(git describe --tags --exact-match 2>/dev/null)
-
-# Check if a tag is checked out
-if [ -n "$tag_name" ]; then
-    echo "Currently checked out tag: $tag_name"
-
-    # Create a release using gh
-    gh release create "$tag_name" \
-        --generate-notes \
-        ribasim_cli_linux.zip \
-        ribasim_cli_windows.zip \
-        ribasim_qgis.zip \
-        generated_testmodels.zip
-
-    echo "Release created successfully."
-
-else
-    echo "No tag is currently checked out."
```

Comment:
Don't you need to check for the "No tag is currently checked out." condition anymore?

Reply:
It's not needed. I tested it locally and git returns with an error code if there's no tag.